src/fetch-wrapper.ts

```diff
@@ -36,23 +36,16 @@ export default function fetchWrapper(
     );
   }
 
-  return fetch(
-    requestOptions.url,
-    Object.assign(
-      {
-        method: requestOptions.method,
-        body: requestOptions.body,
-        headers: requestOptions.headers as HeadersInit,
-        redirect: requestOptions.redirect,
-        // duplex must be set if request.body is ReadableStream or Async Iterables.
-        // See https://fetch.spec.whatwg.org/#dom-requestinit-duplex.
-        ...(requestOptions.body && { duplex: "half" }),
-      },
-      // `requestOptions.request.agent` type is incompatible
-      // see https://github.com/octokit/types.ts/pull/264
-      requestOptions.request as any,
-    ),
-  )
+  return fetch(requestOptions.url, {
+    method: requestOptions.method,
+    body: requestOptions.body,
+    headers: requestOptions.headers as HeadersInit,
+    signal: (requestOptions as any).signal,
+    data: (requestOptions as any).data,
```

Comment:
Did you mean to pass `data` here? Where does that come from?

Reply:
@gr2m I was going off of https://github.com/octokit/request.js/pull/599#issuecomment-1622546734, though looking at it now I forgot the piece about aliasing.

Reply:
@gr2m should @wolfy1339 and I hold off on merging upstream bumps until a new major version has been cut? What do you think would be the best way forward here?

Reply:
It's okay it's just a fix. `options.data` is handled by `@octokit/endpoint`. There is no `options.request.data`. Sorry for the confusion.

Reply:
No problem! It's brought on by my lack of understanding. I've opened https://github.com/octokit/request.js/pull/604 to hopefully correct this.